handshake.md

```diff
@@ -460,13 +461,18 @@ these steps:
 
 3. `WriteMsg(plaintext)`
 
+4. `WriteBytes(0)`
```

Comment:
```suggestion
```

The stream isn't terminated at the end of each message, so I don't believe this belongs here.